### PR TITLE
Add dns policy option

### PR DESCRIPTION
**Pull request by vanhtuan0409**
Add `dnsPolicy` option as it is recommended to use `ClusterFirstWithHostNet` when enable `hostNetwork: true`

https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy

### CODE REVIEW

File: traefik/templates/deployment.yaml [hunk 1]
```diff
@@ -52,6 +52,9 @@ spec:
       serviceAccountName: {{ include "traefik.serviceAccountName" . }}
       terminationGracePeriodSeconds: 60
       hostNetwork: {{ .Values.hostNetwork }}
+      {{- with .Values.deployment.dnsPolicy }}
```
Comment (SantoDE) [inline traefik/templates/deployment.yaml:55]: no values for that?

Comment (vanhtuan0409) [reply traefik/templates/deployment.yaml:55]: Hi, default is we don't need to input any config for `dnsPolicy`

Adding `dnsPolicy: ClusterFirst` might break old deployments

That's why I made it as optional configurable

Comment (SantoDE) [reply traefik/templates/deployment.yaml:55]: which is fine, but I think we should have at least an example in the `values.yml` even if its commented out. Don't you think so?

Comment (vanhtuan0409) [reply traefik/templates/deployment.yaml:55]: @SantoDE Absolutely agree. Added example on `values.yml`

File: traefik/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 8.9.2
+version: 8.9.3
```
Comment (SantoDE) [inline traefik/Chart.yaml:3]: Thanks. Now, last step (sorry) - its adds a new functionality and thus needs to be a minor bump on the version :)

Comment (vanhtuan0409) [reply traefik/Chart.yaml]: @SantoDE no prob. Bumped to 8.11.0.

But shouldn't it be 8.10?

**Pull request merged**